app/controllers/HomeController.scala

```diff
@@ -171,20 +171,23 @@ class HomeController @Inject()(val homeView: views.html.Home,
 }
   }
 
-  private def getDueDates(unpaidCharges: List[FinancialDetailsResponseModel]): List[LocalDate] =
-  (unpaidCharges collect {
-    case fdm: FinancialDetailsModel => fdm.validChargesWithRemainingToPay.getAllDueDates
-  })
-    .flatten
-    .sortWith(_ isBefore _)
-    .sortBy(_.toEpochDay())
+  private def getDueDates(unpaidCharges: List[FinancialDetailsResponseModel], reviewAndReconcileEnabled: Boolean, isFilterOutCodedPoasEnabled: Boolean,
+                          penaltiesEnabled: Boolean)(implicit user: MtdItUser[_]): List[LocalDate] = {
+    val x = unpaidCharges collect {
```

Comment:
can we use more descriptive naming convention here / chargeList or something similar will work well